### ISSUE TITLE
WebSocket support

### ISSUE THREAD

**Issue by venugopal-baswaraju-sony** [sabuhish/fastapi-mqtt#14]
Is web sockets support planned for this library? 
A sample on how to make it work would be highly appreciated

**Comment by sabuhish** [comment]
Yes, we thought about this as well. We did some research if we can have changes to the protocol layer or overwrite it.  This was a tough one though. Not sure yet if we can implement or not, will post here if we can achieve this. Thanks

**Comment by sabuhish** [comment]
Due to the inactivity in this issue, and not planned in the near future, I will close this issue.
